### PR TITLE
Update DG to Vault 1.0

### PR DESCRIPTION
**Pull request by iainthegray**
Updated the Deployment guide with Additional available in Vault 1.0 and bumped product version to 1.0

### CODE REVIEW

File: website/source/guides/operations/deployment-guide.html.md [hunk 1]
```diff
@@ -264,6 +264,17 @@ The `telemetry` stanza specifies various configurations for Vault to publish met
 
 If you decide to configure Vault to publish telemtery data, you should review the [telemetry configuration section](/docs/configuration/telemetry.html) of our documentation.
 
+### High Availability Parameters
+
+New in Vault 1.0 there are additional configuration parameters for HA. Add the below configuration to the Vault configuration file:
+
+```hcl
+api_addr = {{full URL to Vault API endpoint}}
```
Comment (chrishoffman) [inline website/source/guides/operations/deployment-guide.html.md]: This is not a new parameter and has been around since at least 0.8.

Comment (iainthegray) [reply website/source/guides/operations/deployment-guide.html.md]: Thanks - Chris.
It has only now started to warn - hence the update, but I will amend

Comment (iainthegray) [reply website/source/guides/operations/deployment-guide.html.md]: Amended

Comment (roooms) [reply website/source/guides/operations/deployment-guide.html.md]: Looks like there’s a stray bracket in this code example. Should the value be quoted?

**Comment by roooms** [comment]
Looks like there’s a stray bracket in this code example. Should the value be quoted?

**Comment by yhyakuna** [comment]
> Looks like there’s a stray bracket in this code example. Should the value be quoted?

That's a good point.  The HA setup guide does it differently --> https://learn.hashicorp.com/vault/operations/ops-vault-ha-consul#vault-configuration

Should we pick a style?

**Comment by roooms** [comment]
@yhyakuna and @topfunky how do we test these changes now with a local build of middleman redirecting all guides links to learn.hashicorp.com? I'd like to see this change in the context of the whole page being rendered properly.

**Comment by yhyakuna** [comment]
Merged these changes in [learn](https://github.com/hashicorp/learn/blob/master/source/content/vault/advanced/ops-deployment-guide.md) repo and the doc is live now --> https://learn.hashicorp.com/vault/operations/ops-deployment-guide#high-availability-parameters

**Comment by tyrannosaurus-becks** [comment]
@yhyakuna do these still need to be merged here too?

**Pull request merged**